.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
```

Comment:
@romani the latest 3.13.0 (by the way, do you know why 3.x.x? Currently I used 4.6.1) is not working by default. It says -

`Unable to load annotation processing manager org.eclipse.jdt.internal.compiler.apt.dispatch.BatchAnnotationProcessorManager from classpath.`

I've found the only old issue - https://bugs.eclipse.org/bugs/show_bug.cgi?id=507370 , but it is already fixed.